waspc/src/Wasp/Project/ExternalConfig/PackageJson.hs

```diff
@@ -18,21 +17,18 @@ import Wasp.Project.Common
   )
 import qualified Wasp.Util.IO as IOUtil
 
-analyzePackageJsonFile :: Path' Abs (Dir WaspProjectDir) -> IO (Either [String] PackageJson)
-analyzePackageJsonFile waspProjectDir = runExceptT $ do
+readPackageJsonFile :: Path' Abs (Dir WaspProjectDir) -> IO (Either [String] PackageJson)
```

Comment:
You should proably convert the type to `Either String PackageJson` -> doesn't make much sense that it can return mulitple errors.

Same goes for the function below.